Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Action/ActionManagerInterface.h

```diff
@@ -20,6 +20,29 @@ namespace AzToolsFramework
 {
     using ActionManagerOperationResult = AZ::Outcome<void, AZStd::string>;
 
+    struct ActionContextProperties
+    {
+        AZ_RTTI(ActionProperties, "{74694A62-E3FF-43EE-98DF-D66731DC2286}");
+
+        ActionContextProperties() = default;
+        virtual ~ActionContextProperties() = default;
+
+        AZStd::string m_name = "";
+    };
+
+    struct ActionProperties
```

Comment:
These looks great! Thanks for adding 👍 (might be worth an API comment for a little more context but otherwise all good!